charts/matrix-stack/templates/NOTES.txt

```diff
@@ -11,11 +11,21 @@ You are using ESS Community which is tailored to small-/mid-scale, non-commercia
 ESS Pro provides Long-Term Support releases, automatic scaling (horizontal and vertical) of Synapse Workers, Advanced Identity Management to manage users and spaces permissions, and more.
 
 Feel free to join our matrix room to discuss at https://matrix.to/#/#ess-community:element.io.
+
+{{- if $.Values.elementAdmin.enabled }}
+Your admin console is available at: https://{{ $.Values.elementAdmin.ingress.host }}
+{{- if not $.Values.matrixAuthenticationService.enabled }}
```

Comment:
This warning/error shouldn't be present if `$.Values.serverName` isn't set and so the Admin Console is untied from a deployment